flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/planner/plan/utils/RexNodeExtractor.scala

```diff
@@ -538,7 +538,30 @@ class RexNodeToExpressionConverter(
     }
   }
 
-  override def visitFieldAccess(fieldAccess: RexFieldAccess): Option[ResolvedExpression] = None
+  override def visitFieldAccess(fieldAccess: RexFieldAccess): Option[ResolvedExpression] = {
```

Comment:
Sorry, My mistake. This method is called when TableSource implements SupportsFilterPushDown. I will remove this since this patch only supports nested projection pushdown.